src/homepageExperience/containers/CliWizard.tsx

```diff
@@ -0,0 +1,222 @@
+        return (
+          <InitializeClient
+            wizardEventName="cliWizard"
+            setTokenValue={this.setTokenValue}
+            tokenValue={this.state.tokenValue}
+            onSelectBucket={this.handleSelectBucket}
+          />
+        )
+      }
+      case 4: {
+        return <WriteData bucket={this.state.selectedBucket} />
+      }
+      case 5: {
+        return <ExecuteQuery bucket={this.state.selectedBucket} />
+      }
+      case 6: {
+        return <ExecuteAggregateQuery bucket={this.state.selectedBucket} />
+      }
+      case 7: {
+        return (
+          <Finish
+            wizardEventName="cliWizard"
+            markStepAsCompleted={this.handleMarkStepAsCompleted}
+            finishStepCompleted={this.state.finishStepCompleted}
+            finalFeedback={this.state.finalFeedback}
+            setFinalFeedback={this.setFinalFeedback}
+          />
+        )
+      }
+      default: {
+        return <Overview wizard="cliWizard" />
+      }
+    }
+  }
+
+  render() {
+    const {currentStep} = this.state
+
+    return (
+      <Page>
+        <Page.Header fullWidth={false}>
+          {/* Need an empty div so the upgrade button aligns to the right. (Because clockface uses space-between to justifyContent)*/}
+          <div />
+          <RateLimitAlert location="firstMile.homepage" />
+        </Page.Header>
+        <Page.Contents scrollable={true}>
+          <div className="homepage-wizard-container">
+            <aside className="homepage-wizard-container--subway">
+              <div style={{width: '100%'}} data-testid="subway-nav">
+                <SubwayNav
+                  currentStep={this.state.currentStep}
+                  onStepClick={this.handleNavClick}
+                  navigationSteps={HOMEPAGE_NAVIGATION_STEPS_CLI}
+                  settingUpIcon={CLIIcon}
+                  settingUpText="InfluxCLI"
+                  setupTime="5 minutes"
+                />
+              </div>
+            </aside>
+            <div className="homepage-wizard-container--main">
+              <div
+                className={classnames(
+                  'homepage-wizard-container--main-wrapper',
+                  {
+                    verticallyCentered:
+                      this.state.currentStep === 1 ||
+                      this.state.currentStep ===
+                        HOMEPAGE_NAVIGATION_STEPS_CLI.length,
+                  }
+                )}
+              >
+                <WriteDataDetailsContextProvider>
+                  {this.renderStep()}
+                </WriteDataDetailsContextProvider>
+              </div>
+
+              <div className="homepage-wizard-container-footer">
+                <Button
+                  onClick={this.handlePreviousClick}
+                  text="Previous"
+                  size={ComponentSize.Large}
+                  color={ComponentColor.Tertiary}
+                  status={
+                    currentStep > 1
```

Comment:
right now this component mixes using `this.state.currentStep` and this local `currentStep`. I think you should just stick with `this.state.currentStep` and skip storing it into a local variable, since it's already an instance property of the class (i.e. you access it with `this.state.currentStep`